compose/container.py

```diff
@@ -115,6 +115,14 @@ def environment(self):
     def is_running(self):
         return self.get('State.Running')
 
+    @property
+    def cpu_set(self):
+        return self.get('HostConfig.CpusetCpus') or self.get('Config.Cpuset')
```

Comment:
Yes. Reported in https://github.com/docker/docker/issues/13663 and reverted my workarounds.